scripts/assemble_changelog.py

```diff
@@ -0,0 +1,505 @@
+#!/usr/bin/env python3
+
+"""Assemble Mbed TLS change log entries into the change log file.
+
```

Comment:
Is it completely new file in 2.16 or our branch didn't have it before for some reason ?

Reply:
This is a new feature of mainstream Mbed TLS.